completion/kswrapfig.cwl

```diff
@@ -0,0 +1,38 @@
+# kswrapfig package
+# ponte-vecchio 2023-01-17 for v0.004
+
+#keyvals:\usepackage/kswrapfig#c
+insbox
+#endkeyvals
+
+
+\kswrapfig[%<options%>]{%<figure%>}{%<text%>}
+#keyvals:\kswrapfig#c
+Pos=#r,l
+Width=%<dim%>%l
+InPos=#r,c,l
+Sep=%<dim%>%l
+Indent=%<dim%>%l
+Lower=%<dim%>%l
+Caption={%<text%>}
+LastLineSkip=%<dim%>%l
+FirstLineSkip=%<dim%>%l
+List=%<env name%>#itemize,enumerate
```

Comment:
The correct syntax is either `List=%<env name%>` or `List=#itemize,enumerate`

Reply:
This is an immutable list in fact, the documentation specifically lists those two as examples, but it can be any environment name. Should I remove them?

Reply:
Then just `List=%<env name%>`